src/components/Authentication/Login.vue

```diff
@@ -10,25 +10,35 @@
   v-container
     v-form(@submit.prevent="login", ref="login")
       v-text-field(
+          v-if="!otpFormVisible || forceOtp",
           v-model="username",
           label="Username or e-mail",
           autofocus,
           :rules="nonEmptyRules",
           prepend-icon="$vuetify.icons.user",
           type="text")
       v-text-field(
+          v-if="!otpFormVisible || forceOtp",
           v-model="password",
           type="password",
           label="Password",
           :rules="nonEmptyRules",
           prepend-icon="$vuetify.icons.lock")
-      v-layout(row)
+      v-text-field(
+          v-if="otpFormVisible || forceOtp",
+          v-model="otp",
+          type="text",
+          mask="######",
```

Comment:
It's safe to assume this will be 6 digits for the foreseeable future. Nearly all TOTP clients (mobile phone apps, etc.) assume a 6 digit key (with a few supporting 8), so it would break support for many clients to change the OTP length.